src/alias/azext_alias/telemetry/telemetry.py

```diff
@@ -0,0 +1,127 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import datetime
+
+from knack.log import get_logger
+
+from azure.cli.core import telemetry as telemetry_core
+from azure.cli.core.telemetry import _remove_cmd_chars, _remove_symbols, _get_stack_trace
```

Comment:
Why are we importing private methods?
@troydai will we make them public in the future?

Reply:
I'll make a copy of these private methods in `telemetry.py` these private methods import.